test/testrunner.py

```diff
@@ -350,57 +299,18 @@ def main():
                 )
             framework = frameworks_in_images[0]
             is_neuron = "neuron" in dlc_images
-            eks_cluster_name = setup_eks_cluster(framework, is_neuron)
-            configure_eks_cluster(eks_cluster_name)
-
-            if not is_neuron:
-                # setup kubeflow
-                eks_utils.setup_kubeflow(eks_cluster_name)
-
-            # Change 1: Split training and inference, and run one after the other, to prevent scheduling issues
-            # Set -n=4, instead of -n=auto, because initiating too many pods simultaneously has been resulting in
-            # pods timing-out while they were in the Pending state. Scheduling 4 tests (and hence, 4 pods) at once
-            # seems to be an optimal configuration.
-            # Change 2: Separate multi-node EKS tests from single-node tests in execution to prevent resource contention
-            if not is_neuron:
-                pytest_cmds = [
-                    [
-                        "-s",
-                        "-rA",
-                        os.path.join(test_path, framework, "training"),
-                        f"--junitxml={report_train}",
-                        "-n=4",
-                        "-m",
-                        "not multinode",
-                    ],
-                    [
-                        "-s",
-                        "-rA",
-                        os.path.join(test_path, framework, "inference"),
-                        f"--junitxml={report_infer}",
-                        "-n=4",
-                        "-m",
-                        "not multinode",
-                    ],
-                    ["-s", "-rA", test_path, f"--junitxml={report_multinode_train}", "--multinode"],
-                ]
+            build_context = get_build_context()
+            eks_cluster_name = f"{framework}-{build_context}"
```

Comment:
how will this work for things like neuron? will neuron and cpu/gpu tests for tensorflow run on the same cluster? Also will diff versions of a framework run on the same cluster as well?

Reply:
Discussed the same offline